packages/grafana-ui/src/components/Table/TableNG/hooks.test.ts

```diff
@@ -373,14 +374,25 @@ describe('TableNG hooks', () => {
 
     it('should return the calculation if fields array includes this field', () => {
       const { result } = renderHook(() =>
-        useFooterCalcs(rows, [textField, numericField, numericField2], {
+        useFooterCalcs(rows, createDataFrame({ fields: [textField, numericField, numericField2] }), {
           enabled: true,
           footerOptions: { show: true, reducer: ['sum'], fields: ['Field1', 'Field2', 'Field3'] },
         })
       );
 
       expect(result.current).toEqual(['Total', '6', '13']);
     });
+
+    it('should return the calculation if fields array includes this field by either name or display name', () => {
+      const { result } = renderHook(() =>
+        useFooterCalcs(rows, createDataFrame({ fields: [textField, numericField, numericField2] }), {
+          enabled: true,
+          footerOptions: { show: true, reducer: ['sum'], fields: ['Field1', 'Field 2'] },
+        })
+      );
+
+      expect(result.current).toEqual(['Total', '6', '13']);
+    });
```

Comment:
new test confirms the behavior works as expected. I also tested manually by manipulating the panel JSON.